stellarhome/src/main/java/com/stellarhome/controller/RollController.java

```diff
@@ -0,0 +1,10 @@
+package com.stellarhome.controller;
+
+import com.stellarhome.endPoints.IUserEndPoint;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RollController {
```

Comment:
Rename the class to RoleController @DilbaniE

Reply:
change name to RoleController